packages/backend/database/seeders/SubwayStationsToApartmentsSeeder.ts

```diff
@@ -1,18 +1,18 @@
 import BaseSeeder from '@ioc:Adonis/Lucid/Seeder'
 import { faker } from '@faker-js/faker'
 
-import MetroStation from './../../app/Models/MetroStation'
-import Apartment from './../../app/Models/Apartment'
+import MetroStation from 'App/Models/MetroStation'
+import Apartment from 'App/Models/Apartment'
 
 export default class SubwayStationsToApartmentsSeeder extends BaseSeeder {
   public async run (): Promise<void> {
     const аpartments = await Apartment.all()
     const metroStations = await MetroStation.all()
     const metroStationIds = metroStations.map((metroStation) => metroStation.id)
-    const rundomMetroStationId = (): number => { return faker.helpers.arrayElement(Object.values(metroStationIds)) }
+    const rundomMetroStationIds = (): number[] => { return faker.helpers.arrayElements(Object.values(metroStationIds), 2) }
```

Comment:
```suggestion
    const randomMetroStationIds = (): number[] => { return faker.helpers.arrayElements(Object.values(metroStationIds), 2) }
```

VSCode extension: `streetsidesoftware.code-spell-checker`

Reply:
![image](https://user-images.githubusercontent.com/81852394/189108793-7c7707f9-7d2f-42e0-acff-a69714c52d09.png)
Ща исправлю!